geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java

```diff
@@ -2844,41 +2748,22 @@ public String getName() {
     return propertiesList;
   }
 
-  /**
-   * Get the properties for the given declarable.
-   *
-   * @param declarable The declarable
-   * @return Properties found for the given declarable
-   */
   @Override
-  public Properties getDeclarableProperties(final Declarable declarable) {
+  public Properties getDeclarableProperties(Declarable declarable) {
     return declarablePropertiesMap.get(declarable);
   }
 
-  /**
-   * Returns the number of seconds that have elapsed since the Cache was created.
-   *
-   * @since GemFire 3.5
-   */
   @Override
-  public int getUpTime() {
-    return (int) (System.currentTimeMillis() - creationDate.getTime()) / 1000;
+  public long getUpTime() {
```

Comment:
This is internal-only. It's defined on the InternalCache interface. All callers except one use long already. I moved the (int) case to RemoteCacheInfo.